src/utils/milestones.js

```diff
@@ -1,20 +1,16 @@
 export const TimelineData = [
-  {
-    title: '2020 Q4',
-    content: ['Swapr Alpha Launch'],
-    releaseTag: 'ALPHA',
-    past: true,
-  },
   {
     title: '2021 Q1',
     content: [
-      'DXDAO-supplied liquidity',
+      'Swapr Alpha Launch',
+      'DXdao-supplied liquidity',
       'Swapr on xDai',
       'Eco-Routing Beta',
-      'Uniswap, Sushiswap, Honeyswap, LevinSwap Routing',
+      'Uniswap, Suishiswap Honeyswap, LevinSwap Routing',
```

Comment:
I think we need to consider the branding of SushiSwap. The official website says Sushi alone. 

https://www.sushi.com/

cc @0xVenky